streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorTopology.java

```diff
@@ -225,4 +168,8 @@ public String toString(final String indent) {
 
         return Collections.emptySet();
     }
+
+    public List<String> topologicalOrderOfStores() {
+        return stateStores.stream().map(StateStore::name).collect(Collectors.toList());
```

Comment:
`stateStores` is already topologically sorted, so I'm just transforming it to a list of store names.